File: 0151-reverse-words-in-a-string/0151-reverse-words-in-a-string.java
class Solution {
    public String reverseWords(String s) {
        String[] vec = s.trim().split("\\s+");
        String temp="";
        for (int i = vec.length - 1; i > 0; i--){
            temp+=vec[i]+" ";
        }
        return temp+vec[0];
    }
}

File: 0151-reverse-words-in-a-string/0151-reverse-words-in-a-string.cs
public class Solution {
    public string ReverseWords(string s) {
          string[] vec = s.Trim().Split(new[] { ' ' }, StringSplitOptions.RemoveEmptyEntries);
        string temp = "";
        for (int i = vec.Length - 1; i > 0; i--) {
            temp += vec[i] + " ";
        }
        return temp + vec[0];
    }
}